### DISCUSSION TITLE
Changing Folder & File Names Post Creation

### DISCUSSION THREAD

**Discussion by mdchristians** [jmbannon/ytdl-sub#653] [Q&A] [answered]
So I need to change a tv show/playlist's directory names and file names based on data i get from an API call after ytdl-sub downloads the video, but i'm curious if I need to change anything else to prevent the video from being re-downloaded.

**Comment by jmbannon** [comment]
ytdl-sub uses this download archive: https://ytdl-sub.readthedocs.io/en/latest/config.html#ytdl_sub.config.preset_options.OutputOptions.download_archive_name
So long as this file name/path doesn't change, should be good

**Reply by mdchristians** [reply]
I appreciate the quick response!

Unfortunately, they would change.  Would changing the data in those json files work?  Or would they still be re-downloaded

**Reply by jmbannon** [reply, marked as answer]
I meant the file name/path of the archive itself.

The file names and dates in the archive are only used if using https://ytdl-sub.readthedocs.io/en/latest/config.html#ytdl_sub.config.preset_options.OutputOptions.keep_files_after

Otherwise, it only cares about the UUID keys. That tells yt-dlp to not download dupe vids

**Reply by jmbannon** [reply]
I think you should be good

**Reply by mdchristians** [reply]
You rock.  Thank you!
